static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -106,7 +107,7 @@ const getIndexSiteId = (): string => {
         }
     } else {
         const site = config
-            .get('page.pbIndexSites')
+            .get('page.pbIndexSites', [])
```

Comment:
`config.get` takes an optional second parameter that will be used as the fallback if no matches on the first are found. I am setting it to and empty array here, otherwise `config.get` will return `undefined`, and we probably do not want to call `.find` on `undefined` 😸